### PR TITLE
fixing milestones problem

### PR DESCRIPTION
**Pull request by leahbannon**

### CODE REVIEW

File: .about.yml [hunk 1]
```diff
@@ -22,10 +22,10 @@ team:
 - github: noahmanger
 - github: onezerojeremy
 milestones:
-- June 2014: Project Discovery stage started
-  May 2015: Alpha testing of API and website
-  July 2015: Beta testing of the API
-  October 2015: Public release of beta site
+- 'June 2014: Project Discovery stage started'
+- 'May 2015: Alpha testing of API and website'
+- 'July 2015: Beta testing of the API'
```
Comment (noahmanger) [inline .about.yml]: Should this be "Beta release of the API"?

**Comment by mbland** [comment]
Yes, this is the way to go. Any string value with a `:` in it must be enclosed in quotation marks, otherwise YAML will try to parse it as the key of a `key: value` pair.

**Comment by mbland** [comment]
Oh, and I'd merge it for you, but want to wait for @noahmanger's question to get answered.

**Comment by ultrasaurus** [comment]
lets deal with the text change as a separate pull request.  The PR just fixes the technical format

image

Someone from FEC should merge ASAP. cc @leahbannon

**Comment by jmcarp** [comment]
Ping @leahbannon @noahmanger

**Pull request merged**